backend/src/routes/api/nb-events/eventUtils.ts

```diff
@@ -6,12 +6,16 @@ export const getNotebookEvents = async (
   namespace: string,
   nbName: string,
 ): Promise<V1Event[]> => {
-  const eventList = await fastify.kube.coreV1Api
-    .listNamespacedEvent(namespace, undefined, undefined, undefined, 'involvedObject.kind=Pod')
+  return fastify.kube.coreV1Api
+    .listNamespacedEvent(
+      namespace,
+      undefined,
+      undefined,
+      undefined,
+      `involvedObject.kind=Pod,involvedObject.name=${nbName}-0`,
```

Comment:
hmm -- this will probably work for the time being but I feel this is a bit problematic for the future as you're assuming structure in the Kubeflow Notebook pod naming and how that applies to things.

I'm only worried about saying things are ready with another pod still working out things in another spot. But I believe the current infra is just 1 pod, 2 containers (OAuth & Notebook). So we should be fine determining spin up status.

Reply:
Yup, I came up with this after talking with @samuelvl, there's no problem right now, if the name of the pod changes, we'll have a worse problem since we are assuming that structure in several places. Adding the `-0` it's just because we use a StatefulSet, so the name is deterministic, and we are blocking the spawn of more pods for the notebook. 
I agree is not ideal, since we are assuming things and it could break with external changes, but so far I think is one of the best solutions we can have at the moment.